### PR TITLE
Use curl in install instructions + simplify

### PR DESCRIPTION
**Pull request by Samisafool**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,11 +29,9 @@ The required packages are installed during the installation
 ## Installation
 
 #### Download the `install.sh` script, and execute it:
-These commands must be executed in the console. To do this, open the terminal, paste the following lines and press enter:
+This command must be executed in the console. You also need curl installed to use it. To do this, open the terminal, paste the following line and press enter:
 ```
-wget https://github.com/GoXLR-on-Linux/goxlr-on-linux/raw/main/install.sh
-sudo chmod +x ./install.sh
-sudo ./install.sh
+curl -fsSL https://github.com/GoXLR-on-Linux/goxlr-on-linux/raw/main/install.sh | sh
```
Comment (qwertychouskie) [inline README.md:34]: `sh` needs to be `bash`

Comment (Samisafool) [reply README.md:34]: I don't quite understand, but sure. (Please squash my commits when merging)

**Comment by Samisafool** [comment]
Any reason for not merging?

**Comment by lm41** [comment]
Cause it doesn't work with curl:

```
curl -fsSL https://github.com/GoXLR-on-Linux/goxlr-on-linux/raw/main/install.sh | sh                                                                                                   0|INT ✘  8s  
Cloning into 'goxlr-on-linux'...
remote: Enumerating objects: 245, done.
remote: Counting objects: 100% (245/245), done.
remote: Compressing objects: 100% (205/205), done.
remote: Total 245 (delta 135), reused 67 (delta 31), pack-reused 0
Receiving objects: 100% (245/245), 55.59 KiB | 5.56 MiB/s, done.
Resolving deltas: 100% (135/135), done.

GoXLR Full or Mini? Options: full, mini: main: line 41: [: =: unary operator expected
main: line 41: [: =: unary operator expected
 isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: too many arguments
main: line 41: [: too many arguments
#Ask which sound system to use, pulseaudio or pipewire isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: too many arguments
main: line 41: [: too many arguments
#ask_config "type" "Which sound system?" "pulseaudio" "pipewire" isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: =: unary operator expected
main: line 41: [: =: unary operator expected
 isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: too many arguments
main: line 41: [: too many arguments
echo "Working..." isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: =: unary operator expected
main: line 41: [: =: unary operator expected
 isn't an option.
GoXLR Full or Mini? Options: full, mini: #Install isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: too many arguments
main: line 41: [: too many arguments
APT_GET_CMD=$(which apt-get) isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: too many arguments
main: line 41: [: too many arguments
PACMAN_CMD=$(which pacman) isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: =: unary operator expected
main: line 41: [: =: unary operator expected
 isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: too many arguments
main: line 41: [: too many arguments
if [ -n "$APT_GET_CMD" ]; then isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: too many arguments
main: line 41: [: too many arguments
cd $HOME || exit 1 isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: syntax error: `-s' unexpected
main: line 41: [: syntax error: `-s' unexpected
dpkg -s jackd2 >word 2>&1 || sudo apt-get install jackd2 isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: syntax error: `-s' unexpected
main: line 41: [: syntax error: `-s' unexpected
dpkg -s pulseaudio-module-jack >word 2>&1 || sudo apt-get install pulseaudio-module-jack isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: syntax error: `-iq' unexpected
main: line 41: [: syntax error: `-iq' unexpected
grep -iq "source /etc/goxlr/goxlr-on-linux/run_goxlr.sh" .profile || sudo echo "source /etc/goxlr/goxlr-on-linux/run_goxlr.sh" | sudo tee -a ".profile" isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: too many arguments
main: line 41: [: too many arguments
elif [ -n "$PACMAN_CMD" ]; then isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: too many arguments
main: line 41: [: too many arguments
sudo pacman -Qs jack2 || sudo pacman -S jack2 isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: too many arguments
main: line 41: [: too many arguments
sudo pacman -Qs jack2-dbus || sudo pacman -S jack2-dbus isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: too many arguments
main: line 41: [: too many arguments
sudo pacman -Qs pulseaudio-jack || sudo pacman -S pulseaudio-jack isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: too many arguments
main: line 41: [: too many arguments
cd $HOME || exit 1 isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: syntax error: `-iq' unexpected
main: line 41: [: syntax error: `-iq' unexpected
grep -iq "source /etc/goxlr/goxlr-on-linux/run_goxlr.sh" .bash_profile || sudo echo "source /etc/goxlr/goxlr-on-linux/run_goxlr.sh" | sudo tee -a ".bash_profile" isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: too many arguments
main: line 41: [: too many arguments
cd /etc/goxlr/goxlr-on-linux || exit 1 isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: too many arguments
main: line 41: [: too many arguments
sudo cp audio.conf /etc/security/limits.d isn't an option.
GoXLR Full or Mini? Options: full, mini: else isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: too many arguments
main: line 41: [: too many arguments
echo "error can't install packages" isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: too many arguments
main: line 41: [: too many arguments
exit 1; isn't an option.
GoXLR Full or Mini? Options: full, mini: fi isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: =: unary operator expected
main: line 41: [: =: unary operator expected
 isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: too many arguments
main: line 41: [: too many arguments
#Restart PA isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: syntax error: `--kill' unexpected
main: line 41: [: syntax error: `--kill' unexpected
pulseaudio --kill isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: =: unary operator expected
main: line 41: [: =: unary operator expected
 isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: too many arguments
main: line 41: [: too many arguments
#Default cmode for testing both options isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: too many arguments
main: line 41: [: too many arguments
set_config "cmode" "false" isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: =: unary operator expected
main: line 41: [: =: unary operator expected
 isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: too many arguments
main: line 41: [: too many arguments
#Run GoXLR isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: too many arguments
main: line 41: [: too many arguments
sh /etc/goxlr/goxlr-on-linux/run_goxlr.sh|grep "not a valid port" && set_config "cmode" "true" && sh /etc/goxlr/goxlr-on-linux/run_goxlr|grep "not a valid port" && printf "Your GoXLR has been powercycled or was not found.nPlease look in the wiki for other known issues,nif it isn't a know issue Please create one on githubnand attach the GoXLR_Log.txt found in your home directory.n" && sh /etc/goxlr/goxlr-on-linux/genlog.sh isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: =: unary operator expected
main: line 41: [: =: unary operator expected
 isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: =: unary operator expected
main: line 41: [: =: unary operator expected
 isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: too many arguments
main: line 41: [: too many arguments
#clear console isn't an option.
GoXLR Full or Mini? Options: full, mini: clear isn't an option.
GoXLR Full or Mini? Options: full, mini: main: line 41: [: =: unary operator expected
main: line 41: [: =: unary operator expected
 isn't an option.
```
And so on

**Comment by Samisafool** [comment]
Oh yeah, so I'll close this.

**Comment by Samisafool** [comment]
It's weird that it wouldn't work, but eh, whatever

**Pull request closed without merging**